src/osm_configurator/model/project/export.py

```diff
@@ -68,4 +81,5 @@ def export_map(self, path):
         Returns:
             bool: true, if export was successful, otherwise false.
         """
-        pass
+        return self._active_project.get_data_visualizer().create_map(self._active_project.get_config_manager()
```

Comment:
this method shouldn't create a map, the map will be created when someone presses on the button "show map" and it will be saved in the project, you need to find that file and export ist not create a new one.
Maybe we should save where the map is saved? Alternatively search for any file with .html extension in project folder.
You need to handle error correction, what if the file doesn't exists?
Then you need to return a error and view  should be able to create a poup from it.
Weik is responsible for this.